packages/orbit-components/src/Wizard/__tests__/compact.test.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import React from "react";
+import { render, screen, within } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import Wizard, { WizardStep } from "..";
+
+jest.mock("../../hooks/useMediaQuery", () => () => ({ isTablet: false }));
+
+describe("Wizard", () => {
+  describe("compact", () => {
```

Comment:
Seems like redundant describe within describe, you can just write `describe("#Wizard - compact")`, it might be useful if you have several tests inside.